### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by romanbuk52**
add tests CRUD for all methods

### CODE REVIEW

File: cmd/main/main.go [hunk 1]
```diff
@@ -13,20 +13,20 @@ import (
...
 
 	// data := storage.NewDB()
 	data := storageSQL.NewDB(db)
 	handler := web.NewDataHandler(data)
 	router := web.NewPeopleStoreRouter(handler)
-	port := ":8080"
-	server := web.NewServer(port, router)
-	println("server started, port:", port)
+	server := web.NewServer(*port, router)
+	println("server started, port:", *port)
```
Comment (ubombi) [inline cmd/main/main.go:29]: log.Print

File: storageSQL/storageSQL.go [hunk 2]
```diff
@@ -85,27 +85,22 @@ func (d *Data) Get(id string) (m web.Man, er error) {
 func (d *Data) GetAll() (m []web.Man, er error) {
 	readHumans := make([]Human, 0, 10)
 	convertMan := make([]web.Man, 0, 10)
-
-	// println("test1.1", readHumans) // for tests
-	result := d.db.Find(&readHumans)
-	// println("test1.2", readHumans) // for tests
```
Comment (ubombi) [inline storageSQL/storageSQL.go:91]: #deadcode

File: storageSQL/storageSQL.go [hunk 3]
```diff
@@ -85,27 +85,22 @@ func (d *Data) Get(id string) (m web.Man, er error) {
 func (d *Data) GetAll() (m []web.Man, er error) {
 	readHumans := make([]Human, 0, 10)
```
Comment (ubombi) [inline storageSQL/storageSQL.go]: `humans` , `all`, ...

File: storageSQL/storageSQL.go [hunk 4]
```diff
@@ -85,27 +85,22 @@ func (d *Data) Get(id string) (m web.Man, er error) {
...
-	// println("test2.2", convertMan) // for tests
 
-	return convertMan, result.Error
+	return convertMan, res.Error
 }
 
 func (d *Data) Edit(m web.Man) error {
 	newHuman := fromWebModel(m)
-	result := d.db.Model(&newHuman).Updates(newHuman)
+	res := d.db.Model(&newHuman).Updates(newHuman)
```
Comment (ubombi) [inline storageSQL/storageSQL.go]: `newHuman` in edit?

File: web/crud.go [hunk 5]
```diff
@@ -52,15 +52,18 @@ func (dh *DataHandler) GetAllMan(w http.ResponseWriter, r *http.Request) {
 	}
 	//							<<
 	if err := json.NewEncoder(w).Encode(data); err != nil {
-		dh.SendError(w, http.StatusInternalServerError, ErrJsonEncode)
+		dh.SendError(w, http.StatusInternalServerError, err)
```
Comment (ubombi) [inline web/crud.go:55]: why?

File: web/crud.go [hunk 6]
```diff
@@ -52,15 +52,18 @@ func (dh *DataHandler) GetAllMan(w http.ResponseWriter, r *http.Request) {
...
 	}
 }
 
 // GetManByID get man by ID
 func (dh *DataHandler) GetManByID(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
-	id := vars["manID"]
+	id, ok := vars["manID"]
+	if !ok {
+		dh.SendError(w, http.StatusBadRequest, ErrManNotFound)
```
Comment (ubombi) [inline web/crud.go]: ~~ErrManNotFound~~ => ErrBadRequest.
Not found is different one

**Review by ubombi — APPROVED** [review]
overall LGTM

File: web/crud.go (hunk 5 above)
Comment (romanbuk52) [reply web/crud.go:55]: that would I see an error with Encoder.

**Pull request merged**